nextcord/guild.py

```diff
@@ -627,6 +627,16 @@ def categories(self) -> List[CategoryChannel]:
         r.sort(key=lambda c: (c.position, c.id))
         return r
 
+    @property
+    def forum_channels(self) -> List[ForumChannel]:
+        """List[:class:`ForumChannel`]: A list of forum channels that belongs to this guild.
```

Comment:
```diff
- A list of forum channels that belongs to this guild.
+ A list of forum channels that belong to this guild.
```

Reply:
Sorry about this, only once 
```suggestion
        """List[:class:`ForumChannel`]: A list of forum channels that belong to this guild.
```
please use suggestion blocks (file icon with `+` and `-` symbols at the top of the comment box) so they're commitable directly.